ipaserver/install/server/replicainstall.py

```diff
@@ -747,6 +786,10 @@ def install_check(installer):
                          ldap_uri=ldapuri)
     remote_api.finalize()
     installer._remote_api = remote_api
+
+    with rpc_client(remote_api) as client:
+        check_remote_fips_mode(client, api.env.fips_mode)
```

Comment:
The DL1 twice checking remark is false, thought this code was in `common_check()`. The other part still applies, though.